src/query/api/v1/handler/prometheus/native/common.go

```diff
@@ -128,38 +136,47 @@ func parseParams(r *http.Request) (models.RequestParams, *handler.ParseError) {
 	return params, nil
 }
 
-func parseTarget(r *http.Request) (string, error) {
-	targetQueries, ok := r.URL.Query()[targetParam]
-	if !ok || len(targetQueries) == 0 || targetQueries[0] == "" {
+func parseQuery(r *http.Request) (string, error) {
+	queries, ok := r.URL.Query()[queryParam]
+	if !ok || len(queries) == 0 || queries[0] == "" {
 		return "", errors.ErrNoTargetFound
```

Comment:
nit: rename `ErrNoTargetFound` to `ErrNoQueryFound`?

Reply:
Sounds good.